tools/spack-scripting/scripting/cmd/devbuildcosmo.py

```diff
@@ -119,9 +120,17 @@ def devbuildcosmo(self, args):
     # Clean if needed
     if args.clean_build:
         print("\033[92m" + "==> " + "\033[0m" + "cosmo: Cleaning build directory")
-        subprocess.run(["make", "clean"], cwd=source_path + "/cosmo/ACC")
+        files_to_clean = []
 
-        if os.path.exists(base_directory + "/spack-build"):
+        # make clean, but without make command
```

Comment:
why not using make clean ? This is implementing 2 time the same behviour in Makefile and in spack (what if we change the name of the intermediate claw file directory in the cosmo makefie).
Alternative solution:
- fix the cosmo Makefile
- set F90 before calling make clean, ex: F90=gfortran make clean (this is the easiest) . It may even work by setting F90="NOTSET" in the make line

Reply:
I juste tested this workaround:
`F90=NOT_SET make clean`
this works, so this shoudl also work:
        `subprocess.run(["F90=NOTSET","make", "clean"], cwd=source_path + "/cosmo/ACC")`

Reply:
setting F90 via `subprocess.run`does not work. I used `os.environ["F90"]` instead.

Reply:
If you use os.environ this may have side effect - because F90 is now set globally - My proposition was to have it on the same line so that it will only be set locally for the make clean command. One solution would be to unset  it after the make clean then if there is no way to do the equivalent of `F90=NOT_SET make clean` . I don't know the os command for that.

Reply:
`os.environ.pop` does that. Trying to acces F90 after the pop leads to an error -> assume the env variable is not set anymore